support-files/mini-benchmark.sh

```diff
@@ -240,3 +286,6 @@ case $RESULT in
     fi
     ;;
 esac
+# Record the output into the log file, if requested
+) 2>&1 | ($LOG && tee "$BENCHMARK_NAME"-"$TIMESTAMP".log)
```

Comment:
easier way:
```
exec 2>&1 | ($LOG && tee "$BENCHMARK_NAME"-"$TIMESTAMP".log)
```
At the place where the `(` begins.

Reply:
If I understand correctly, the suggestion is to remove the [outermost `(...)` wrapper (at L99)](https://github.com/MariaDB/server/pull/2550/files/362529acac0ac9b030a47dd9c9d9f61179c923d7#diff-31ffc6ead737b0fde857304ec2501c45b01753115079125a5a9f55f0d7ec4c55R99) and instead use `exec` to redirect the output cleanly.

Using the exact `exec` line above, the output was not being logged to the file. The following worked instead, but readability is slightly worse:
```bash
exec > >($LOG && tee "$BENCHMARK_NAME"-"$TIMESTAMP".log) 2>&1
```

There are other ways of achieving the same result, though I prefer the original implementation (using the [outermost `(...)` wrapper](https://github.com/MariaDB/server/pull/2550/files/362529acac0ac9b030a47dd9c9d9f61179c923d7#diff-31ffc6ead737b0fde857304ec2501c45b01753115079125a5a9f55f0d7ec4c55R99)); it makes it clear what is being logged (everything within the `(...)` block) and the redirection/logging statement is readable.